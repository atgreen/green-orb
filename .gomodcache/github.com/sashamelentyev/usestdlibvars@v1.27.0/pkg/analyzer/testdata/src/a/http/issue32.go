// Code generated by usestdlibvars, DO NOT EDIT.

package http_test

import "net/http"

var (
	_ = "connect"
	_ = "delete"
	_ = "get"
	_ = "head"
	_ = "options"
	_ = "patch"
	_ = "post"
	_ = "put"
	_ = "trace"
)

const (
	_ = "connect"
	_ = "delete"
	_ = "get"
	_ = "head"
	_ = "options"
	_ = "patch"
	_ = "post"
	_ = "put"
	_ = "trace"
)

var (
	_ = http.Request{Method: "connect"} // want `"connect" can be replaced by http\.MethodConnect`
	_ = http.Request{Method: "delete"}  // want `"delete" can be replaced by http\.MethodDelete`
	_ = http.Request{Method: "get"}     // want `"get" can be replaced by http\.MethodGet`
	_ = http.Request{Method: "head"}    // want `"head" can be replaced by http\.MethodHead`
	_ = http.Request{Method: "options"} // want `"options" can be replaced by http\.MethodOptions`
	_ = http.Request{Method: "patch"}   // want `"patch" can be replaced by http\.MethodPatch`
	_ = http.Request{Method: "post"}    // want `"post" can be replaced by http\.MethodPost`
	_ = http.Request{Method: "put"}     // want `"put" can be replaced by http\.MethodPut`
	_ = http.Request{Method: "trace"}   // want `"trace" can be replaced by http\.MethodTrace`
)

var (
	_ = &http.Request{Method: "connect"} // want `"connect" can be replaced by http\.MethodConnect`
	_ = &http.Request{Method: "delete"}  // want `"delete" can be replaced by http\.MethodDelete`
	_ = &http.Request{Method: "get"}     // want `"get" can be replaced by http\.MethodGet`
	_ = &http.Request{Method: "head"}    // want `"head" can be replaced by http\.MethodHead`
	_ = &http.Request{Method: "options"} // want `"options" can be replaced by http\.MethodOptions`
	_ = &http.Request{Method: "patch"}   // want `"patch" can be replaced by http\.MethodPatch`
	_ = &http.Request{Method: "post"}    // want `"post" can be replaced by http\.MethodPost`
	_ = &http.Request{Method: "put"}     // want `"put" can be replaced by http\.MethodPut`
	_ = &http.Request{Method: "trace"}   // want `"trace" can be replaced by http\.MethodTrace`
)

var (
	_, _ = http.NewRequest("connect", "", http.NoBody) // want `"connect" can be replaced by http\.MethodConnect`
	_, _ = http.NewRequest("delete", "", http.NoBody)  // want `"delete" can be replaced by http\.MethodDelete`
	_, _ = http.NewRequest("get", "", http.NoBody)     // want `"get" can be replaced by http\.MethodGet`
	_, _ = http.NewRequest("head", "", http.NoBody)    // want `"head" can be replaced by http\.MethodHead`
	_, _ = http.NewRequest("options", "", http.NoBody) // want `"options" can be replaced by http\.MethodOptions`
	_, _ = http.NewRequest("patch", "", http.NoBody)   // want `"patch" can be replaced by http\.MethodPatch`
	_, _ = http.NewRequest("post", "", http.NoBody)    // want `"post" can be replaced by http\.MethodPost`
	_, _ = http.NewRequest("put", "", http.NoBody)     // want `"put" can be replaced by http\.MethodPut`
	_, _ = http.NewRequest("trace", "", http.NoBody)   // want `"trace" can be replaced by http\.MethodTrace`
)

var (
	_, _ = http.NewRequestWithContext(nil, "connect", "", http.NoBody) // want `"connect" can be replaced by http\.MethodConnect`
	_, _ = http.NewRequestWithContext(nil, "delete", "", http.NoBody)  // want `"delete" can be replaced by http\.MethodDelete`
	_, _ = http.NewRequestWithContext(nil, "get", "", http.NoBody)     // want `"get" can be replaced by http\.MethodGet`
	_, _ = http.NewRequestWithContext(nil, "head", "", http.NoBody)    // want `"head" can be replaced by http\.MethodHead`
	_, _ = http.NewRequestWithContext(nil, "options", "", http.NoBody) // want `"options" can be replaced by http\.MethodOptions`
	_, _ = http.NewRequestWithContext(nil, "patch", "", http.NoBody)   // want `"patch" can be replaced by http\.MethodPatch`
	_, _ = http.NewRequestWithContext(nil, "post", "", http.NoBody)    // want `"post" can be replaced by http\.MethodPost`
	_, _ = http.NewRequestWithContext(nil, "put", "", http.NoBody)     // want `"put" can be replaced by http\.MethodPut`
	_, _ = http.NewRequestWithContext(nil, "trace", "", http.NoBody)   // want `"trace" can be replaced by http\.MethodTrace`
)
