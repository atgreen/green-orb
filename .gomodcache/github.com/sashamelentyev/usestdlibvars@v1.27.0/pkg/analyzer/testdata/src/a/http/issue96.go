package http_test

import (
	"net/http"
)

func _() error {
	resp, err := http.DefaultClient.Do(&http.Request{})
	if err != nil {
		return err
	}
	defer func() { _ = resp.Body.Close() }()

	if resp.StatusCode/100 != 2 {
		return nil
	}
	if resp.StatusCode+100 != 2 {
		return nil
	}
	if resp.StatusCode-100 != 2 {
		return nil
	}
	if resp.StatusCode*100 != 2 {
		return nil
	}

	return nil
}
