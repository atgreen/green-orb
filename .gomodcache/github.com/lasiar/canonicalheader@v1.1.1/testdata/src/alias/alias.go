package alias

import "net/http"

type myHeader = http.Header

func _() {
	myHeader{}.Get("TT") // want `non-canonical header "TT", instead use: "Tt"`
}
