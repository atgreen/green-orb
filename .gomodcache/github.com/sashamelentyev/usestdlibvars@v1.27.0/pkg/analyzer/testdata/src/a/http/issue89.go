// Code generated by usestdlibvars, DO NOT EDIT.

package http_test

import "net/http"

func _() error {
	resp, err := http.DefaultClient.Do(&http.Request{})
	if err != nil {
		return err
	}
	defer func() { _ = resp.Body.Close() }()
	if resp.StatusCode < 100 {
		return nil
	} else if resp.StatusCode < 100 {
		return nil
	}
	if resp.StatusCode < 101 {
		return nil
	} else if resp.StatusCode < 101 {
		return nil
	}
	if resp.StatusCode < 102 {
		return nil
	} else if resp.StatusCode < 102 {
		return nil
	}
	if resp.StatusCode < 103 {
		return nil
	} else if resp.StatusCode < 103 {
		return nil
	}
	if resp.StatusCode < 200 {
		return nil
	} else if resp.StatusCode < 200 {
		return nil
	}
	if resp.StatusCode < 201 {
		return nil
	} else if resp.StatusCode < 201 {
		return nil
	}
	if resp.StatusCode < 202 {
		return nil
	} else if resp.StatusCode < 202 {
		return nil
	}
	if resp.StatusCode < 203 {
		return nil
	} else if resp.StatusCode < 203 {
		return nil
	}
	if resp.StatusCode < 204 {
		return nil
	} else if resp.StatusCode < 204 {
		return nil
	}
	if resp.StatusCode < 205 {
		return nil
	} else if resp.StatusCode < 205 {
		return nil
	}
	if resp.StatusCode < 206 {
		return nil
	} else if resp.StatusCode < 206 {
		return nil
	}
	if resp.StatusCode < 207 {
		return nil
	} else if resp.StatusCode < 207 {
		return nil
	}
	if resp.StatusCode < 208 {
		return nil
	} else if resp.StatusCode < 208 {
		return nil
	}
	if resp.StatusCode < 226 {
		return nil
	} else if resp.StatusCode < 226 {
		return nil
	}
	if resp.StatusCode < 300 {
		return nil
	} else if resp.StatusCode < 300 {
		return nil
	}
	if resp.StatusCode < 301 {
		return nil
	} else if resp.StatusCode < 301 {
		return nil
	}
	if resp.StatusCode < 302 {
		return nil
	} else if resp.StatusCode < 302 {
		return nil
	}
	if resp.StatusCode < 303 {
		return nil
	} else if resp.StatusCode < 303 {
		return nil
	}
	if resp.StatusCode < 304 {
		return nil
	} else if resp.StatusCode < 304 {
		return nil
	}
	if resp.StatusCode < 305 {
		return nil
	} else if resp.StatusCode < 305 {
		return nil
	}
	if resp.StatusCode < 307 {
		return nil
	} else if resp.StatusCode < 307 {
		return nil
	}
	if resp.StatusCode < 308 {
		return nil
	} else if resp.StatusCode < 308 {
		return nil
	}
	if resp.StatusCode < 400 {
		return nil
	} else if resp.StatusCode < 400 {
		return nil
	}
	if resp.StatusCode < 401 {
		return nil
	} else if resp.StatusCode < 401 {
		return nil
	}
	if resp.StatusCode < 402 {
		return nil
	} else if resp.StatusCode < 402 {
		return nil
	}
	if resp.StatusCode < 403 {
		return nil
	} else if resp.StatusCode < 403 {
		return nil
	}
	if resp.StatusCode < 404 {
		return nil
	} else if resp.StatusCode < 404 {
		return nil
	}
	if resp.StatusCode < 405 {
		return nil
	} else if resp.StatusCode < 405 {
		return nil
	}
	if resp.StatusCode < 406 {
		return nil
	} else if resp.StatusCode < 406 {
		return nil
	}
	if resp.StatusCode < 407 {
		return nil
	} else if resp.StatusCode < 407 {
		return nil
	}
	if resp.StatusCode < 408 {
		return nil
	} else if resp.StatusCode < 408 {
		return nil
	}
	if resp.StatusCode < 409 {
		return nil
	} else if resp.StatusCode < 409 {
		return nil
	}
	if resp.StatusCode < 410 {
		return nil
	} else if resp.StatusCode < 410 {
		return nil
	}
	if resp.StatusCode < 411 {
		return nil
	} else if resp.StatusCode < 411 {
		return nil
	}
	if resp.StatusCode < 412 {
		return nil
	} else if resp.StatusCode < 412 {
		return nil
	}
	if resp.StatusCode < 413 {
		return nil
	} else if resp.StatusCode < 413 {
		return nil
	}
	if resp.StatusCode < 414 {
		return nil
	} else if resp.StatusCode < 414 {
		return nil
	}
	if resp.StatusCode < 415 {
		return nil
	} else if resp.StatusCode < 415 {
		return nil
	}
	if resp.StatusCode < 416 {
		return nil
	} else if resp.StatusCode < 416 {
		return nil
	}
	if resp.StatusCode < 417 {
		return nil
	} else if resp.StatusCode < 417 {
		return nil
	}
	if resp.StatusCode < 418 {
		return nil
	} else if resp.StatusCode < 418 {
		return nil
	}
	if resp.StatusCode < 421 {
		return nil
	} else if resp.StatusCode < 421 {
		return nil
	}
	if resp.StatusCode < 422 {
		return nil
	} else if resp.StatusCode < 422 {
		return nil
	}
	if resp.StatusCode < 423 {
		return nil
	} else if resp.StatusCode < 423 {
		return nil
	}
	if resp.StatusCode < 424 {
		return nil
	} else if resp.StatusCode < 424 {
		return nil
	}
	if resp.StatusCode < 425 {
		return nil
	} else if resp.StatusCode < 425 {
		return nil
	}
	if resp.StatusCode < 426 {
		return nil
	} else if resp.StatusCode < 426 {
		return nil
	}
	if resp.StatusCode < 428 {
		return nil
	} else if resp.StatusCode < 428 {
		return nil
	}
	if resp.StatusCode < 429 {
		return nil
	} else if resp.StatusCode < 429 {
		return nil
	}
	if resp.StatusCode < 431 {
		return nil
	} else if resp.StatusCode < 431 {
		return nil
	}
	if resp.StatusCode < 451 {
		return nil
	} else if resp.StatusCode < 451 {
		return nil
	}
	if resp.StatusCode < 500 {
		return nil
	} else if resp.StatusCode < 500 {
		return nil
	}
	if resp.StatusCode < 501 {
		return nil
	} else if resp.StatusCode < 501 {
		return nil
	}
	if resp.StatusCode < 502 {
		return nil
	} else if resp.StatusCode < 502 {
		return nil
	}
	if resp.StatusCode < 503 {
		return nil
	} else if resp.StatusCode < 503 {
		return nil
	}
	if resp.StatusCode < 504 {
		return nil
	} else if resp.StatusCode < 504 {
		return nil
	}
	if resp.StatusCode < 505 {
		return nil
	} else if resp.StatusCode < 505 {
		return nil
	}
	if resp.StatusCode < 506 {
		return nil
	} else if resp.StatusCode < 506 {
		return nil
	}
	if resp.StatusCode < 507 {
		return nil
	} else if resp.StatusCode < 507 {
		return nil
	}
	if resp.StatusCode < 508 {
		return nil
	} else if resp.StatusCode < 508 {
		return nil
	}
	if resp.StatusCode < 510 {
		return nil
	} else if resp.StatusCode < 510 {
		return nil
	}
	if resp.StatusCode < 511 {
		return nil
	} else if resp.StatusCode < 511 {
		return nil
	}
	if resp.StatusCode > 100 {
		return nil
	} else if resp.StatusCode > 100 {
		return nil
	}
	if resp.StatusCode > 101 {
		return nil
	} else if resp.StatusCode > 101 {
		return nil
	}
	if resp.StatusCode > 102 {
		return nil
	} else if resp.StatusCode > 102 {
		return nil
	}
	if resp.StatusCode > 103 {
		return nil
	} else if resp.StatusCode > 103 {
		return nil
	}
	if resp.StatusCode > 200 {
		return nil
	} else if resp.StatusCode > 200 {
		return nil
	}
	if resp.StatusCode > 201 {
		return nil
	} else if resp.StatusCode > 201 {
		return nil
	}
	if resp.StatusCode > 202 {
		return nil
	} else if resp.StatusCode > 202 {
		return nil
	}
	if resp.StatusCode > 203 {
		return nil
	} else if resp.StatusCode > 203 {
		return nil
	}
	if resp.StatusCode > 204 {
		return nil
	} else if resp.StatusCode > 204 {
		return nil
	}
	if resp.StatusCode > 205 {
		return nil
	} else if resp.StatusCode > 205 {
		return nil
	}
	if resp.StatusCode > 206 {
		return nil
	} else if resp.StatusCode > 206 {
		return nil
	}
	if resp.StatusCode > 207 {
		return nil
	} else if resp.StatusCode > 207 {
		return nil
	}
	if resp.StatusCode > 208 {
		return nil
	} else if resp.StatusCode > 208 {
		return nil
	}
	if resp.StatusCode > 226 {
		return nil
	} else if resp.StatusCode > 226 {
		return nil
	}
	if resp.StatusCode > 300 {
		return nil
	} else if resp.StatusCode > 300 {
		return nil
	}
	if resp.StatusCode > 301 {
		return nil
	} else if resp.StatusCode > 301 {
		return nil
	}
	if resp.StatusCode > 302 {
		return nil
	} else if resp.StatusCode > 302 {
		return nil
	}
	if resp.StatusCode > 303 {
		return nil
	} else if resp.StatusCode > 303 {
		return nil
	}
	if resp.StatusCode > 304 {
		return nil
	} else if resp.StatusCode > 304 {
		return nil
	}
	if resp.StatusCode > 305 {
		return nil
	} else if resp.StatusCode > 305 {
		return nil
	}
	if resp.StatusCode > 307 {
		return nil
	} else if resp.StatusCode > 307 {
		return nil
	}
	if resp.StatusCode > 308 {
		return nil
	} else if resp.StatusCode > 308 {
		return nil
	}
	if resp.StatusCode > 400 {
		return nil
	} else if resp.StatusCode > 400 {
		return nil
	}
	if resp.StatusCode > 401 {
		return nil
	} else if resp.StatusCode > 401 {
		return nil
	}
	if resp.StatusCode > 402 {
		return nil
	} else if resp.StatusCode > 402 {
		return nil
	}
	if resp.StatusCode > 403 {
		return nil
	} else if resp.StatusCode > 403 {
		return nil
	}
	if resp.StatusCode > 404 {
		return nil
	} else if resp.StatusCode > 404 {
		return nil
	}
	if resp.StatusCode > 405 {
		return nil
	} else if resp.StatusCode > 405 {
		return nil
	}
	if resp.StatusCode > 406 {
		return nil
	} else if resp.StatusCode > 406 {
		return nil
	}
	if resp.StatusCode > 407 {
		return nil
	} else if resp.StatusCode > 407 {
		return nil
	}
	if resp.StatusCode > 408 {
		return nil
	} else if resp.StatusCode > 408 {
		return nil
	}
	if resp.StatusCode > 409 {
		return nil
	} else if resp.StatusCode > 409 {
		return nil
	}
	if resp.StatusCode > 410 {
		return nil
	} else if resp.StatusCode > 410 {
		return nil
	}
	if resp.StatusCode > 411 {
		return nil
	} else if resp.StatusCode > 411 {
		return nil
	}
	if resp.StatusCode > 412 {
		return nil
	} else if resp.StatusCode > 412 {
		return nil
	}
	if resp.StatusCode > 413 {
		return nil
	} else if resp.StatusCode > 413 {
		return nil
	}
	if resp.StatusCode > 414 {
		return nil
	} else if resp.StatusCode > 414 {
		return nil
	}
	if resp.StatusCode > 415 {
		return nil
	} else if resp.StatusCode > 415 {
		return nil
	}
	if resp.StatusCode > 416 {
		return nil
	} else if resp.StatusCode > 416 {
		return nil
	}
	if resp.StatusCode > 417 {
		return nil
	} else if resp.StatusCode > 417 {
		return nil
	}
	if resp.StatusCode > 418 {
		return nil
	} else if resp.StatusCode > 418 {
		return nil
	}
	if resp.StatusCode > 421 {
		return nil
	} else if resp.StatusCode > 421 {
		return nil
	}
	if resp.StatusCode > 422 {
		return nil
	} else if resp.StatusCode > 422 {
		return nil
	}
	if resp.StatusCode > 423 {
		return nil
	} else if resp.StatusCode > 423 {
		return nil
	}
	if resp.StatusCode > 424 {
		return nil
	} else if resp.StatusCode > 424 {
		return nil
	}
	if resp.StatusCode > 425 {
		return nil
	} else if resp.StatusCode > 425 {
		return nil
	}
	if resp.StatusCode > 426 {
		return nil
	} else if resp.StatusCode > 426 {
		return nil
	}
	if resp.StatusCode > 428 {
		return nil
	} else if resp.StatusCode > 428 {
		return nil
	}
	if resp.StatusCode > 429 {
		return nil
	} else if resp.StatusCode > 429 {
		return nil
	}
	if resp.StatusCode > 431 {
		return nil
	} else if resp.StatusCode > 431 {
		return nil
	}
	if resp.StatusCode > 451 {
		return nil
	} else if resp.StatusCode > 451 {
		return nil
	}
	if resp.StatusCode > 500 {
		return nil
	} else if resp.StatusCode > 500 {
		return nil
	}
	if resp.StatusCode > 501 {
		return nil
	} else if resp.StatusCode > 501 {
		return nil
	}
	if resp.StatusCode > 502 {
		return nil
	} else if resp.StatusCode > 502 {
		return nil
	}
	if resp.StatusCode > 503 {
		return nil
	} else if resp.StatusCode > 503 {
		return nil
	}
	if resp.StatusCode > 504 {
		return nil
	} else if resp.StatusCode > 504 {
		return nil
	}
	if resp.StatusCode > 505 {
		return nil
	} else if resp.StatusCode > 505 {
		return nil
	}
	if resp.StatusCode > 506 {
		return nil
	} else if resp.StatusCode > 506 {
		return nil
	}
	if resp.StatusCode > 507 {
		return nil
	} else if resp.StatusCode > 507 {
		return nil
	}
	if resp.StatusCode > 508 {
		return nil
	} else if resp.StatusCode > 508 {
		return nil
	}
	if resp.StatusCode > 510 {
		return nil
	} else if resp.StatusCode > 510 {
		return nil
	}
	if resp.StatusCode > 511 {
		return nil
	} else if resp.StatusCode > 511 {
		return nil
	}
	if resp.StatusCode <= 100 {
		return nil
	} else if resp.StatusCode < 100 {
		return nil
	}
	if resp.StatusCode <= 101 {
		return nil
	} else if resp.StatusCode < 101 {
		return nil
	}
	if resp.StatusCode <= 102 {
		return nil
	} else if resp.StatusCode < 102 {
		return nil
	}
	if resp.StatusCode <= 103 {
		return nil
	} else if resp.StatusCode < 103 {
		return nil
	}
	if resp.StatusCode <= 200 {
		return nil
	} else if resp.StatusCode < 200 {
		return nil
	}
	if resp.StatusCode <= 201 {
		return nil
	} else if resp.StatusCode < 201 {
		return nil
	}
	if resp.StatusCode <= 202 {
		return nil
	} else if resp.StatusCode < 202 {
		return nil
	}
	if resp.StatusCode <= 203 {
		return nil
	} else if resp.StatusCode < 203 {
		return nil
	}
	if resp.StatusCode <= 204 {
		return nil
	} else if resp.StatusCode < 204 {
		return nil
	}
	if resp.StatusCode <= 205 {
		return nil
	} else if resp.StatusCode < 205 {
		return nil
	}
	if resp.StatusCode <= 206 {
		return nil
	} else if resp.StatusCode < 206 {
		return nil
	}
	if resp.StatusCode <= 207 {
		return nil
	} else if resp.StatusCode < 207 {
		return nil
	}
	if resp.StatusCode <= 208 {
		return nil
	} else if resp.StatusCode < 208 {
		return nil
	}
	if resp.StatusCode <= 226 {
		return nil
	} else if resp.StatusCode < 226 {
		return nil
	}
	if resp.StatusCode <= 300 {
		return nil
	} else if resp.StatusCode < 300 {
		return nil
	}
	if resp.StatusCode <= 301 {
		return nil
	} else if resp.StatusCode < 301 {
		return nil
	}
	if resp.StatusCode <= 302 {
		return nil
	} else if resp.StatusCode < 302 {
		return nil
	}
	if resp.StatusCode <= 303 {
		return nil
	} else if resp.StatusCode < 303 {
		return nil
	}
	if resp.StatusCode <= 304 {
		return nil
	} else if resp.StatusCode < 304 {
		return nil
	}
	if resp.StatusCode <= 305 {
		return nil
	} else if resp.StatusCode < 305 {
		return nil
	}
	if resp.StatusCode <= 307 {
		return nil
	} else if resp.StatusCode < 307 {
		return nil
	}
	if resp.StatusCode <= 308 {
		return nil
	} else if resp.StatusCode < 308 {
		return nil
	}
	if resp.StatusCode <= 400 {
		return nil
	} else if resp.StatusCode < 400 {
		return nil
	}
	if resp.StatusCode <= 401 {
		return nil
	} else if resp.StatusCode < 401 {
		return nil
	}
	if resp.StatusCode <= 402 {
		return nil
	} else if resp.StatusCode < 402 {
		return nil
	}
	if resp.StatusCode <= 403 {
		return nil
	} else if resp.StatusCode < 403 {
		return nil
	}
	if resp.StatusCode <= 404 {
		return nil
	} else if resp.StatusCode < 404 {
		return nil
	}
	if resp.StatusCode <= 405 {
		return nil
	} else if resp.StatusCode < 405 {
		return nil
	}
	if resp.StatusCode <= 406 {
		return nil
	} else if resp.StatusCode < 406 {
		return nil
	}
	if resp.StatusCode <= 407 {
		return nil
	} else if resp.StatusCode < 407 {
		return nil
	}
	if resp.StatusCode <= 408 {
		return nil
	} else if resp.StatusCode < 408 {
		return nil
	}
	if resp.StatusCode <= 409 {
		return nil
	} else if resp.StatusCode < 409 {
		return nil
	}
	if resp.StatusCode <= 410 {
		return nil
	} else if resp.StatusCode < 410 {
		return nil
	}
	if resp.StatusCode <= 411 {
		return nil
	} else if resp.StatusCode < 411 {
		return nil
	}
	if resp.StatusCode <= 412 {
		return nil
	} else if resp.StatusCode < 412 {
		return nil
	}
	if resp.StatusCode <= 413 {
		return nil
	} else if resp.StatusCode < 413 {
		return nil
	}
	if resp.StatusCode <= 414 {
		return nil
	} else if resp.StatusCode < 414 {
		return nil
	}
	if resp.StatusCode <= 415 {
		return nil
	} else if resp.StatusCode < 415 {
		return nil
	}
	if resp.StatusCode <= 416 {
		return nil
	} else if resp.StatusCode < 416 {
		return nil
	}
	if resp.StatusCode <= 417 {
		return nil
	} else if resp.StatusCode < 417 {
		return nil
	}
	if resp.StatusCode <= 418 {
		return nil
	} else if resp.StatusCode < 418 {
		return nil
	}
	if resp.StatusCode <= 421 {
		return nil
	} else if resp.StatusCode < 421 {
		return nil
	}
	if resp.StatusCode <= 422 {
		return nil
	} else if resp.StatusCode < 422 {
		return nil
	}
	if resp.StatusCode <= 423 {
		return nil
	} else if resp.StatusCode < 423 {
		return nil
	}
	if resp.StatusCode <= 424 {
		return nil
	} else if resp.StatusCode < 424 {
		return nil
	}
	if resp.StatusCode <= 425 {
		return nil
	} else if resp.StatusCode < 425 {
		return nil
	}
	if resp.StatusCode <= 426 {
		return nil
	} else if resp.StatusCode < 426 {
		return nil
	}
	if resp.StatusCode <= 428 {
		return nil
	} else if resp.StatusCode < 428 {
		return nil
	}
	if resp.StatusCode <= 429 {
		return nil
	} else if resp.StatusCode < 429 {
		return nil
	}
	if resp.StatusCode <= 431 {
		return nil
	} else if resp.StatusCode < 431 {
		return nil
	}
	if resp.StatusCode <= 451 {
		return nil
	} else if resp.StatusCode < 451 {
		return nil
	}
	if resp.StatusCode <= 500 {
		return nil
	} else if resp.StatusCode < 500 {
		return nil
	}
	if resp.StatusCode <= 501 {
		return nil
	} else if resp.StatusCode < 501 {
		return nil
	}
	if resp.StatusCode <= 502 {
		return nil
	} else if resp.StatusCode < 502 {
		return nil
	}
	if resp.StatusCode <= 503 {
		return nil
	} else if resp.StatusCode < 503 {
		return nil
	}
	if resp.StatusCode <= 504 {
		return nil
	} else if resp.StatusCode < 504 {
		return nil
	}
	if resp.StatusCode <= 505 {
		return nil
	} else if resp.StatusCode < 505 {
		return nil
	}
	if resp.StatusCode <= 506 {
		return nil
	} else if resp.StatusCode < 506 {
		return nil
	}
	if resp.StatusCode <= 507 {
		return nil
	} else if resp.StatusCode < 507 {
		return nil
	}
	if resp.StatusCode <= 508 {
		return nil
	} else if resp.StatusCode < 508 {
		return nil
	}
	if resp.StatusCode <= 510 {
		return nil
	} else if resp.StatusCode < 510 {
		return nil
	}
	if resp.StatusCode <= 511 {
		return nil
	} else if resp.StatusCode < 511 {
		return nil
	}
	if resp.StatusCode >= 100 {
		return nil
	} else if resp.StatusCode > 100 {
		return nil
	}
	if resp.StatusCode >= 101 {
		return nil
	} else if resp.StatusCode > 101 {
		return nil
	}
	if resp.StatusCode >= 102 {
		return nil
	} else if resp.StatusCode > 102 {
		return nil
	}
	if resp.StatusCode >= 103 {
		return nil
	} else if resp.StatusCode > 103 {
		return nil
	}
	if resp.StatusCode >= 200 {
		return nil
	} else if resp.StatusCode > 200 {
		return nil
	}
	if resp.StatusCode >= 201 {
		return nil
	} else if resp.StatusCode > 201 {
		return nil
	}
	if resp.StatusCode >= 202 {
		return nil
	} else if resp.StatusCode > 202 {
		return nil
	}
	if resp.StatusCode >= 203 {
		return nil
	} else if resp.StatusCode > 203 {
		return nil
	}
	if resp.StatusCode >= 204 {
		return nil
	} else if resp.StatusCode > 204 {
		return nil
	}
	if resp.StatusCode >= 205 {
		return nil
	} else if resp.StatusCode > 205 {
		return nil
	}
	if resp.StatusCode >= 206 {
		return nil
	} else if resp.StatusCode > 206 {
		return nil
	}
	if resp.StatusCode >= 207 {
		return nil
	} else if resp.StatusCode > 207 {
		return nil
	}
	if resp.StatusCode >= 208 {
		return nil
	} else if resp.StatusCode > 208 {
		return nil
	}
	if resp.StatusCode >= 226 {
		return nil
	} else if resp.StatusCode > 226 {
		return nil
	}
	if resp.StatusCode >= 300 {
		return nil
	} else if resp.StatusCode > 300 {
		return nil
	}
	if resp.StatusCode >= 301 {
		return nil
	} else if resp.StatusCode > 301 {
		return nil
	}
	if resp.StatusCode >= 302 {
		return nil
	} else if resp.StatusCode > 302 {
		return nil
	}
	if resp.StatusCode >= 303 {
		return nil
	} else if resp.StatusCode > 303 {
		return nil
	}
	if resp.StatusCode >= 304 {
		return nil
	} else if resp.StatusCode > 304 {
		return nil
	}
	if resp.StatusCode >= 305 {
		return nil
	} else if resp.StatusCode > 305 {
		return nil
	}
	if resp.StatusCode >= 307 {
		return nil
	} else if resp.StatusCode > 307 {
		return nil
	}
	if resp.StatusCode >= 308 {
		return nil
	} else if resp.StatusCode > 308 {
		return nil
	}
	if resp.StatusCode >= 400 {
		return nil
	} else if resp.StatusCode > 400 {
		return nil
	}
	if resp.StatusCode >= 401 {
		return nil
	} else if resp.StatusCode > 401 {
		return nil
	}
	if resp.StatusCode >= 402 {
		return nil
	} else if resp.StatusCode > 402 {
		return nil
	}
	if resp.StatusCode >= 403 {
		return nil
	} else if resp.StatusCode > 403 {
		return nil
	}
	if resp.StatusCode >= 404 {
		return nil
	} else if resp.StatusCode > 404 {
		return nil
	}
	if resp.StatusCode >= 405 {
		return nil
	} else if resp.StatusCode > 405 {
		return nil
	}
	if resp.StatusCode >= 406 {
		return nil
	} else if resp.StatusCode > 406 {
		return nil
	}
	if resp.StatusCode >= 407 {
		return nil
	} else if resp.StatusCode > 407 {
		return nil
	}
	if resp.StatusCode >= 408 {
		return nil
	} else if resp.StatusCode > 408 {
		return nil
	}
	if resp.StatusCode >= 409 {
		return nil
	} else if resp.StatusCode > 409 {
		return nil
	}
	if resp.StatusCode >= 410 {
		return nil
	} else if resp.StatusCode > 410 {
		return nil
	}
	if resp.StatusCode >= 411 {
		return nil
	} else if resp.StatusCode > 411 {
		return nil
	}
	if resp.StatusCode >= 412 {
		return nil
	} else if resp.StatusCode > 412 {
		return nil
	}
	if resp.StatusCode >= 413 {
		return nil
	} else if resp.StatusCode > 413 {
		return nil
	}
	if resp.StatusCode >= 414 {
		return nil
	} else if resp.StatusCode > 414 {
		return nil
	}
	if resp.StatusCode >= 415 {
		return nil
	} else if resp.StatusCode > 415 {
		return nil
	}
	if resp.StatusCode >= 416 {
		return nil
	} else if resp.StatusCode > 416 {
		return nil
	}
	if resp.StatusCode >= 417 {
		return nil
	} else if resp.StatusCode > 417 {
		return nil
	}
	if resp.StatusCode >= 418 {
		return nil
	} else if resp.StatusCode > 418 {
		return nil
	}
	if resp.StatusCode >= 421 {
		return nil
	} else if resp.StatusCode > 421 {
		return nil
	}
	if resp.StatusCode >= 422 {
		return nil
	} else if resp.StatusCode > 422 {
		return nil
	}
	if resp.StatusCode >= 423 {
		return nil
	} else if resp.StatusCode > 423 {
		return nil
	}
	if resp.StatusCode >= 424 {
		return nil
	} else if resp.StatusCode > 424 {
		return nil
	}
	if resp.StatusCode >= 425 {
		return nil
	} else if resp.StatusCode > 425 {
		return nil
	}
	if resp.StatusCode >= 426 {
		return nil
	} else if resp.StatusCode > 426 {
		return nil
	}
	if resp.StatusCode >= 428 {
		return nil
	} else if resp.StatusCode > 428 {
		return nil
	}
	if resp.StatusCode >= 429 {
		return nil
	} else if resp.StatusCode > 429 {
		return nil
	}
	if resp.StatusCode >= 431 {
		return nil
	} else if resp.StatusCode > 431 {
		return nil
	}
	if resp.StatusCode >= 451 {
		return nil
	} else if resp.StatusCode > 451 {
		return nil
	}
	if resp.StatusCode >= 500 {
		return nil
	} else if resp.StatusCode > 500 {
		return nil
	}
	if resp.StatusCode >= 501 {
		return nil
	} else if resp.StatusCode > 501 {
		return nil
	}
	if resp.StatusCode >= 502 {
		return nil
	} else if resp.StatusCode > 502 {
		return nil
	}
	if resp.StatusCode >= 503 {
		return nil
	} else if resp.StatusCode > 503 {
		return nil
	}
	if resp.StatusCode >= 504 {
		return nil
	} else if resp.StatusCode > 504 {
		return nil
	}
	if resp.StatusCode >= 505 {
		return nil
	} else if resp.StatusCode > 505 {
		return nil
	}
	if resp.StatusCode >= 506 {
		return nil
	} else if resp.StatusCode > 506 {
		return nil
	}
	if resp.StatusCode >= 507 {
		return nil
	} else if resp.StatusCode > 507 {
		return nil
	}
	if resp.StatusCode >= 508 {
		return nil
	} else if resp.StatusCode > 508 {
		return nil
	}
	if resp.StatusCode >= 510 {
		return nil
	} else if resp.StatusCode > 510 {
		return nil
	}
	if resp.StatusCode >= 511 {
		return nil
	} else if resp.StatusCode > 511 {
		return nil
	}
	return nil
}
