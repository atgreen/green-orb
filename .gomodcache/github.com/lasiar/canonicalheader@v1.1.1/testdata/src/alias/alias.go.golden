package alias

import "net/http"

type myHeader = http.Header

func _() {
	myHeader{}.Get("Tt") // want `non-canonical header "TT", instead use: "Tt"`
}
